interop-testing/src/main/java/io/grpc/testing/integration/AbstractNettyInteropTest.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2020 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.testing.integration;
+
+import io.grpc.netty.InternalNettyServerBuilder;
+import io.grpc.netty.NettyServerBuilder;
+
+public abstract class AbstractNettyInteropTest extends AbstractInteropTest {
```

Comment:
If `customCensusModulePresent()` was not necessarily (because the default matches the stats boolean), then it would seem appropriate to remove this class and inline the calls to setStatsEnabled and addStreamTracerFactory; that's similar to what is being done on client-side.

Even right now it's not clear this class is adding enough value for the complexity. _Especially_ for the okhttp test extending it; it would be much better to duplicate to avoid the problem of needing to extend two classes and just to reduce surprise.

Reply:
Great idea.